primitives/primproc/primitiveserver.cpp

```diff
@@ -1598,7 +1598,7 @@ struct BPPHandler
   {
     uint32_t uniqueID, sessionID, stepID;
     BPPMap::iterator it;
-
+#if 0
```

Comment:
You surely don't need this preprocessor if

Reply:
You are right, I do not.